src/webparts/controlsTest/components/TestControl.tsx

```diff
@@ -1,37 +1,39 @@
-import * as React from "react";
+import * as React from 'react';
```

Comment:
Can you try to revert the formatting changes?

Reply:
done